js/pro/krakenfutures.js

```diff
@@ -0,0 +1,387 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const krakenfuturesRest = require ('../krakenfutures.js');
+const { BadSymbol, BadRequest } = require ('../base/errors');
+const { ArrayCache, ArrayCacheBySymbolById, ArrayCacheByTimestamp } = require ('./base/Cache');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class krakenfutures extends krakenfuturesRest {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'has': {
+                'ws': true,
+                'watchBalance': false, // for now
+                'watchMyTrades': false, // for now
+                'watchOHLCV': false, // for now
+                'watchOrderBook': false, // for now
+                'watchOrders': false, // for now
+                'watchTicker': true,
+                'watchTickers': false, // for now
+                'watchTrades': false, // for now
+                // 'watchHeartbeat': true,
+                // 'watchStatus': true,
+            },
+            'urls': {
+                'api': {
+                    'ws': {
+                        'public': 'wss://futures.kraken.com/ws/v1',
+                    },
```

Comment:
`krakenfutures` has only one ws endpoint.

```suggestion
                    'ws': 'wss://futures.kraken.com/ws/v1',
```

I found other exchanges simply do like this

Reply:
@youknowone yeah we can do that